README.md

```diff
@@ -42,24 +42,12 @@ oc new-project <namespace>
 
 - Some products will need AWS credentials so create 2 secrets in the Namespace/Project for the Integreatly Operator
     ```
-    apiVersion: v1
-    kind: Secret
-    metadata:
-      name: s3-credentials
-      namespace: <installation-namespace>
-    stringData:
-      AWS_ACCESS_KEY_ID: <your-aws-access-key>
-      AWS_SECRET_ACCESS_KEY: <your-aws-secret-key>
-    ```
-    ```
-    apiVersion: v1
-    kind: Secret
-    metadata:
-      name: s3-bucket
-      namespace: <installation-namespace>
-    stringData:
-      AWS_BUCKET: <an-aws-s3-bucket-name>
-      AWS_REGION: <region-s3-bucket-name-is-in>
+   oc process -f https://gist.githubusercontent.com/philbrookes/b35a9571f326fcdeb8e3270772a35839/raw/b7ac6da42c7f576cebe82fbf3a2dfa1ef5974e5d/gistfile1.txt \
```

Comment:
Looks good to me, but should this be moved to the deploy dir in the repo?

Reply:
I also don't like that 'master' branch points to philbrookes gist. I would prefer the gist content to be moved to the repo directly (deploy dir sound good to me)

Reply:
good idea, I've updated it.